mythril/interfaces/cli.py

```diff
@@ -17,8 +17,8 @@
 from mythril.version import VERSION
 
 
-def exit_with_error(format, message):
-    if format == 'text' or format == 'markdown':
+def exit_with_error(fmt, message):
```

Comment:
Why this change? Isn't format more readable in this case?

Reply:
In this case format is also the name of a python built-in function: https://docs.python.org/3/library/functions.html#format

Reply:
Can we do format_ vs fmt?

Reply:
or how about `format_type`

Reply:
Not saying its the optimal solution. But in my defense, using a trailing underscore is the pep8 proposed way to prevent shadowing: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

Reply:
Wow, didn't know that! Well, we (usually) shouldn't argue with PEP8, so I'll change that back with the rest of the additions. Thanks for going out of your way to look that up :+1:

Reply:
I'm totally fine with format_type in this case :smile:

Reply:
Too late. Both solutions are equally good anyway and just as readable. :laughing: